easybuild/easyconfigs/n/nodejs/nodejs-14.17.0-GCCcore-10.3.0.eb

```diff
@@ -2,7 +2,7 @@ easyblock = 'ConfigureMake'
 
 name = 'nodejs'
 version = '14.17.0'  # LTS on 2021-06-03
-local_libversion = '83'
+_libversion = '83'
```

Comment:
@lexming why this change?

Reply:
I systematically remove `local` from custom variables in easyconfigs since the change allowing `_variable` as name. It's a better format.

Reply:
I think we're pretty consistent in using `local_*` for local variables in easyconfigs, so we should stick to that in the central easyconfigs repo, I think... "`It's a better format`" is highly subjective, no?

That said, it's not like we should be looking for reasons to keep PRs open longer either.

So as long as we don't enforce either `_*` or `local_*` in the tests, this shouldn't block a PR from being merged.

Reply:
It's not wasted effort, because I think we should actually enforce the use of `local_*` in the easyconfig test suite.

EasyBuild allows using either `local_*` or `_*`, but in the central easyconfigs repo it makes sense to be a bit more strict to keep some consistency across easyconfigs we include. Similar, in some sense, to how we require `use_pip`, and restrict dependency versions in a particular easyconfig generation.

Reply:
Some "stats": there's currently 205 easyconfigs that use `_*` for local variables, and 3910 that use `local_*`.

So it's not consistent currently, but making it consistent should definitely go towards using `local_*` everywhere instead of `_*`.

Reply:
@boegel any ideas on the ratio for recent (new) contributions? From my experience, a lot of the `local_` use is in bumped older ECs